pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixInstanceDataManager.java

```diff
@@ -84,6 +91,22 @@ public class HelixInstanceDataManager implements InstanceDataManager {
   private static final Logger LOGGER = LoggerFactory.getLogger(HelixInstanceDataManager.class);
 
   private final Map<String, TableDataManager> _tableDataManagerMap = new ConcurrentHashMap<>();
+
+  // A cache for the logical table context
+  private static final String TABLE_CONFIG_PATH_PREFIX = "/CONFIGS/TABLE/";
```

Comment:
Can these constants be shared with other classes ? I remember it has been declared in at least one more place.

Reply:
Yes. They are. Should I use the CommonConstants class to define these?